facebook/OpenBIC
# fby3.5: hd: Support GPIO interrupt

Summary:
- Support monitor system DC state.
- Support monitor post complete.

Dependency: #382 

Test plan:
- Build code: PASS
- Check system DC flag will change: Pass

Log:
- power on
```
BMC:
root@bmc-oob:~# power-util slot1 on
Powering fru 1 to ON state...
root@bmc-oob:~# power-util slot1 status
Power status for fru 1 : ON
BIC:
uart:~$ [set_DC_status] gpio number(39) status(1)
[set_post_status] gpio number(1) status(1)
```
- power off
```
BMC:
root@bmc-oob:~# power-util slot1 off
Powering fru 1 to OFF state...
root@bmc-oob:~# power-util slot1 status
Power status for fru 1 : OFF
BIC:
uart:~$ [set_post_status] gpio number(1) status(0)
[set_DC_status] gpio number(39) status(0)
```
